docs/markdown/snippets/add_str_splitlines_method.md

```diff
@@ -0,0 +1,3 @@
+## Added str.splitlines method
+
+`str.splitlines()` can now be used to split a string into an array of lines.
```

Comment:
```suggestion
[[str.splitlines]] can now be used to split a string into an array of lines.
```

This should cause our inter-links extension to make this a hyperlink to the new method.